### PR TITLE
Add .module extension to SCSS files

### LINKED ISSUE #1511: Rename SCSS files to include .module before file extension
# Issue Description
Tools like webpacker and create-react-app v2 enable CSS modules automatically for stylesheet files that include `.module` before the extension name. `file.module.css`, `another-file.module.scss`
This would allow terra components to be used more easily with these tools with less config needed to enable CSS modules.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Files that end in `.scss` are renamed to end in `.module.scss`

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Resolves #1511

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,6 +81,7 @@
     "react-dom": "^16.2.0",
     "react-router-dom": "^4.0.0",
     "react-test-renderer": "^16.2.0",
+    "replace-in-file": "^3.4.0",
```
Comment (mjhenkes) [inline package.json]: Could this be removed?

Comment (bjankord) [reply package.json]: Ah yes, need to remove that file, I use it for doing mass changes to the changelog, but I forgot it was still in the package.json file.

Comment (bjankord) [reply package.json]: Removed in this commit: https://github.com/cerner/terra-core/pull/1717/commits/d70e6d2f491dfc308024101b48040fae3db3834b

File: packages/terra-time-input/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Changelog
 Unreleased
 ----------
 ### Changed
+* Add .module extension to SCSS file(s)
+### Changed
```
Comment (emilyrohrbough) [inline packages/terra-time-input/CHANGELOG.md]: whoops. Extra `### Changed`

Comment (bjankord) [reply packages/terra-time-input/CHANGELOG.md]: Updated: https://github.com/cerner/terra-core/pull/1717/commits/a2c2969c79be590764cf58bf1852c44fb782f9b4

**Pull request merged**